app/src/main/java/waiter/Reactor/Reactor.java

```diff
@@ -0,0 +1,18 @@
+package waiter.Reactor;
+
+import waiter.Interactor.Interactive;
+import waiter.Interactor.Interactor;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public record Reactor(ServerSocket serverSocket) implements Reactive {
```

Comment:
This is the interface of the wrapper (Reactor) for the Socket Java Class

Reactive was the name chosen as this is what reacts to the users input, it is the response point to communicate with the user and give a response (whatever that response may be, in this case echoing back the user input)